internal/metrics/metricsManager.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-//go:generate mockgen -source=metricsManager.go -package=metrics -destination=mock_netricsManager_api.go
+//go:generate mockgen -source=metricsManager.go -package=metrics -destination=mock_metricsManager_api.go
```

Comment:
what is this strange combination of snake case and camel case? 
```suggestion
//go:generate mockgen -source=metricsManager.go -package=metrics -destination=mock_metrics_manager_api.go
```

Reply:
This is not related to the PR, I just fixed the typo `netricsManager` --> `metricsManager`.

Reply:
but you already changed it so why not fix it?

Reply:
done.